libopenfpga/libpinconstrain/src/file_readers/pcf_reader.h

```diff
@@ -0,0 +1,37 @@
+#ifndef PCF_READER_H
+#define PCF_READER_H
+
+#include <iostream>
+#include <string>
+#include <sstream>
+#include <fstream>
+#include <vector>
+#include <unordered_map>
+#include <map>
+
+/*
+Supported PCF commands:
+
+* set_io  <net> <pad> - constrain a given <net> to a given physical <pad> in eFPGA pinout.
+* set_clk <pin> <net> - constrain a given global clock <pin> to a given <net>
+
+  Every tile where <net> is present will be constrained to use a given global clock.
+*/
+using namespace std;
```

Comment:
Better to avoid the use of ``std`` here. We always specify the namespace when calling each object, e.g., std::vector<>

Reply:
Fixed